dev/app/builtin/stories/analytics.stories.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/default */
+
+import { action, storiesOf } from 'dev-novel';
+import instantsearch from '../../../../index';
+import { wrapWithHits } from '../../utils/wrap-with-hits.js';
+
+const stories = storiesOf('Analytics');
+
+export default () => {
```

Comment:
Why does the code export a function?

Reply:
to not have side-effect on import maybe?

Reply:
Actually, I just reproduce the same pattern than before. But good question @bobylito , and after some research @Haroenv has right. Since we have different kind of stories (builtin, jquery, vanilla) if we don't apply them explicitly, they will all be loaded by default.